op-supervisor/supervisor/backend/db/events.go

```diff
@@ -0,0 +1 @@
+package db
```

Comment:
What is this empty file for? Can we delete it?

Reply:
yes, leftover from a previous experiment